pandas/core/generic.py

```diff
@@ -3277,16 +3280,19 @@ def to_latex(
             Character recognized as decimal separator, e.g. ',' in Europe.
         multicolumn : bool, default True
             Use \multicolumn to enhance MultiIndex columns.
-            The default will be read from the config module.
+            The default will be read from the config module, and is set
+            as the option ``styler.sparse.columns``.
         multicolumn_format : str, default 'l'
             The alignment for multicolumns, similar to `column_format`
-            The default will be read from the config module.
+            The default will be read from the config module, and is set as the option
+            ``styler.latex.multicol_align``.
         multirow : bool, default False
```

Comment:
For the arguments that changed defaults, I think the `default` here needs to change. 

Could you also add a `..versionchange:: 2.0.0` directive in the body describing that the default changed?

Reply:
will do good spot,

Reply:
Done and test passing. Few failures in CIArm are unrelated.